src/cgame/cg_main.cpp

```diff
@@ -1184,6 +1185,8 @@ void CG_Init( int serverMessageNum, int clientNum, const glconfig_t& gl, const G
 	new(&cg) cg_t{};
 	for ( centity_t& ent : cg_entities) { ent = {}; }
 
+	CG_RmlUIDebuggerDisable(); // CHECKME
```

Comment:
CG_Init is only called after a VM is freshly loaded so this is not needed here. Maybe you meant to put it in a shutdown function?

Reply:
Further, I think rmlui will clean up the debugger by itself in the shutdown function. Thus, I think we can probably remove the function all together.